$OpenBSD: patch-Source_NSObject_m,v 1.10 2011/05/24 13:58:28 sebastia Exp $

fix building with clang, not needed yet, but to not forget about it

--- Source/NSObject.m.orig	Thu Apr 14 14:36:06 2011
+++ Source/NSObject.m	Mon May 23 12:57:03 2011
@@ -955,7 +955,7 @@ objc_create_block_classes_as_subclasses_of(Class super
       finalize_sel = @selector(finalize);
       finalize_imp = class_getMethodImplementation(self, finalize_sel);
 
-#if (defined(__FreeBSD__) || defined(__OpenBSD__)) && defined(__i386__)
+#if defined(__FreeBSD__) && defined(__i386__)
       // Manipulate the FPU to add the exception mask. (Fixes SIGFPE
       // problems on *BSD)
       // Note this only works on x86
